force-app/main/default/classes/SOQL.cls

```diff
@@ -248,7 +286,9 @@ public with sharing class SOQL {
      * @return The Account Name of the Contact.
      */
     public static String getAccountNameForContact(String contactId) {
-        return null; // Replace null with the variable you used to store the result
+        List<Contact> con  = [SELECT Account.Name FROM Contact WHERE Id = :contactId LIMIT 1];
+        String accountName = con[0].Account.Name;
```

Comment:
For practice try looping over the list of contacts even though it's only 1 and setting he account name